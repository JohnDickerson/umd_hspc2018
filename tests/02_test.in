4
100 10 10
123 123 123
123 123 5
100 10 10
