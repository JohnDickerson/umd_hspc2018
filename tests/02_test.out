49 71 86 14 76 34 94 26 91 27
1 2 4 7 11 16 22 29 37 46 56 67 79 92 106 121 20 40 60 81 102 5 33 61 88 116 30 64 97 13 52 91 12 57 101 28 77 6 63 114 53 111 54 117 68 14 84 38 110 73 34 115 85 51 26 122 100 86 72 62 50 48 49 59 71 83 99 120 25 65 98 19 74 123 66 9 87 39 8 104 89 76 75 82 103 3 36 94 27 107 58 42 35 44 80 119 69 24 18 31 78 17 113 15 70 43 90 23 45 10 93 105 109 21 96 118 55 112 47 32 108 41 95
94 40 31 46 107
49 71 86 14 76 34 94 26 91 27
